client/lib/plans/features-list.tsx

```diff
@@ -311,6 +310,8 @@ import {
 	FEATURE_SENSEI_STORAGE,
 	FEATURE_SENSEI_HOSTING,
 	FEATURE_SENSEI_JETPACK,
+	WPCOM_FEATURES_PREMIUM_THEMES_LIMITED,
+	WPCOM_FEATURES_PREMIUM_THEMES_UNLIMITED,
```

Comment:
I'm wondering if we could slightly change the order of Limited/Unlimited within the feature.

I'm thinking about:
- `WPCOM_FEATURES_LIMITED_PREMIUM_THEMES`
- `WPCOM_FEATURES_UNLIMITED_PREMIUM_THEMES`

What do you think?

Reply:
I think I still prefer to have a "variant" suffix at the end (reminds me of the BEM notation for CSS), at least for me it's more clear this way. But I'm not strongly attached to that, so I'm happy to change them if you think we should.

Reply:
No strong opinion here so let's keep your implementation.